File: src/main/java/com/trading/kalyani/KTManager/model/IndexOHLC.java
package com.trading.kalyani.KTManager.model;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class IndexOHLC {

    Integer appJobConfigNum;

    String indexLTP;

    String thresholdValue;

    String dayLow;

    String dayHigh;


}

File: kpn-ui/src/app/models/IndexOHLC.ts
export class IndexOHLC {
    appJobConfigNum: number;
    indexLTP: string;
    thresholdValue: string;
    dayLow: string;
    dayHigh: string;
  
    constructor(
      appJobConfigNum: number,
      indexLTP: string,
      thresholdValue: string,
      dayLow: string,
      dayHigh: string 
    ) {
      this.appJobConfigNum = appJobConfigNum;
      this.indexLTP = indexLTP;
      this.thresholdValue = thresholdValue;
      this.dayLow = dayLow;
      this.dayHigh = dayHigh;
    }
  }